### ISSUE TITLE
Script crashes when submitting a POST request

### ISSUE THREAD

**Issue by SimonEast** [jamonkko/console-log-server#1]
This is just the tool I've been looking for, thanks! I've been trying to find a way to debug outgoing API calls from my web app, and this looks really helpful.

Unfortunately the console-log-server keeps crashing when I send it a POST request using curl (either via command line or PHP-curl). Example:

    curl --data "test=1" localhost:3000

Here is the crash message. Do you have any idea about what might be causing this?  It seems to be that the script cannot correctly determine the `bodyType` and so it throws an exception.

I installed the app from npm, if that makes a difference.

```
console-log-server listening on http://localhost:3000

*************************
POST /
headers:
  host:           localhost:3000
  user-agent:     curl/7.47.0
  accept:         */*
  content-length: 10
  content-type:   application/x-www-form-urlencoded
query: (empty)
C:\Dev\____\node_modules\console-log-server\dist\logging.js:94
      throw new Error('Internal Error! bodyType not set!');
      ^

Error: Internal Error! bodyType not set!
    at exports.default (C:\Dev\______\node_modules\console-log-server\dist\logging.js:94:13)
    at ServerResponse.<anonymous> (C:\Dev\______\node_modules\console-log-server\dist\router.js:79:29)
    at emitNone (events.js:91:20)
    at ServerResponse.emit (events.js:185:7)
    at finish (_http_outgoing.js:593:10)
    at afterWrite (_stream_writable.js:388:3)
    at _combinedTickCallback (internal/process/next_tick.js:80:20)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by jamonkko** [comment]
Thanks for reporting this! 

I noticed that I had forgot to publish support for url encoded forms, and that commit cdbb307c5e784ed479d3125fdc8b55485f784d8a just sat there waiting and would have prevented this error :( 

I published 0.1.4 now that should work for you. And great if this tool is helpful. I wrote it originally because (for big surprise) was not able to find simplistic one from npm.

**Comment by SimonEast** [comment]
Oh wow, thanks so much. That's great. Hope to give it another try later
this week. And yeah, was surprising nobody else had done something like
this.

Thanks again.
